ios/RNSScreenStack.mm

```diff
@@ -656,9 +645,12 @@ - (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer
       return NO;
     }
     [self cancelTouchesInParent];
+#ifdef RN_FABRIC_ENABLED
+    return _controller.viewControllers.count >= 2;
```

Comment:
Why is it needed if we have this code ?
```
if (![topScreen isKindOfClass:[RNSScreenView class]] || !topScreen.gestureEnabled ||
      _controller.viewControllers.count < 2) {
    return NO;
  }
```

Reply:
Good catch

[be15f17](https://github.com/software-mansion/react-native-screens/pull/1434/commits/be15f17dce706957b142e5c9794ae6500c430aab)